start/sw/web-app/laravel5.md

```diff
@@ -9,41 +9,49 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
+
 
 #### Install MariaDB Relational Database Server
 Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
+Or you can install by : 
+```bash
+# install MariaDB
+$ dnf install mariadb-server mariadb
+# start MariaDB
+$ systemctl enable --now mysql
+# initialize MariaDB
+$ sudo mysql_secure_installation
+```
 
-#### Install PHP Extensions
+#### Install PHP and all its dependencies
+Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
+##### Fedora 28 uses 7.2 by default
 ```bash
-# Laravel required PHP extensions.
-$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
-# Restart Apache
-$ sudo systemctl restart httpd
+# It will install all required extensions, perk of Fedora
+$ sudo dnf install composer
 ```
 
-#### Install Composer
-We will need composer to install laravel and it's dependencies
+#### Install Valet Linux
+We will need to set it up for easier development
```

Comment:
What is Valet? Maybe you could give a bit more info on what is it and why you need it instead of making the user jump to Laravel website.